src/debugger/server/channel/DebugChannel.js

```diff
@@ -75,8 +75,8 @@ export class DebugChannel {
     this.writeOut(this._marshaller.marshallBreakpointAcknowledge(requestID, messageType, args.breakpoints));
   }
 
-  sendStoppedResponse(reason: StoppedReason, filePath: string, line: number, column: number): void {
-    this.writeOut(this._marshaller.marshallStoppedResponse(reason, filePath, line, column));
+  sendStoppedResponse(reason: StoppedReason, filePath: string, line: number, column: number, message?: string): void {
```

Comment:
Do you want to marshall message as undefined if it is not supplied? Consider making the parameter not optional, or provide a more useful default.

Reply:
The message should be optional, since it is only used by debugger Diagnostic messages and nothing else. The most appropriate default message for non-diagnostic cases would be nothing ("") -- should I go with that? That solution collides with a previous comment of yours:
> Better yet, provide a default message that is not blank.

This message also can't be combined into the `reason` field, since `StoppedReason` is a flow enumeration of strings -- we can't enumerate all possible error messages.

Reply:
In the light of this, it seems OK to make message optional.